scripts/build-default.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
+# This file is part of vs-fltk
+# License: https://github.com/KaruroChori/vs-fltk/blob/master/LICENCE.md
+
+set -e
+
+# Ensure the script is run from the source root
+if [ ! -f "src/app/main.cpp" ]; then
+  echo "This script must be run from the source root."
+  exit 1
+fi
+
+set -v
+if [ "CI" != "true" ]; then
```

Comment:
I added this section to update subprojects as a separate commit. Let me know if you hate it and I'll revert it. ;) I liked this idea better than the prompt w/timeout.